internal/envoy/manager/cache_manager.go

```diff
@@ -44,7 +44,7 @@ type cacheManager struct {
 func NewCacheManager(snapshotCache cache_v3.SnapshotCache, logger logr.Logger) *cacheManager {
 	return &cacheManager{
 		SnapshotCache: snapshotCache,
-		fleetSnapshot: make(map[string]*cache_v3.Snapshot),
+		fleetSnapshot: map[string]*cache_v3.Snapshot{},
```

Comment:
Why did this need to change?

Reply:
Just made it shorter.

I can revert it, if you want?

Reply:
revert it

Reply:
Reverted. Thanks.